examples/controls/webpack.config.js

```diff
@@ -0,0 +1,65 @@
+// NOTE: This is a Webpack 2 configuration file for react-map-gl
+const {resolve} = require('path');
+const webpack = require('webpack');
+
+module.exports = {
+  // Example entry point
+  entry: {
+    app: resolve('./app.js')
+  },
+
+  // Silence excessive webpack dev server warnings
+  devServer: {
+    stats: {
+      warnings: false
+    }
+  },
+
+  devtool: 'source-maps',
+
+  resolve: {
+    alias: {
+      // Work against the latest base library in this repo
+      'react-map-gl': resolve('../..'),
```

Comment:
If we haven't already copied the local development webpack setup of deck and luma here, we should. For most users the example is most valuable if it works against an installed version as it can then be copied and used outside of this repo.